website/client-old/js/components/groupTasks/groupTasksController.js

```diff
@@ -158,10 +158,12 @@ habitrpg.controller('GroupTasksCtrl', ['$scope', 'Shared', 'Tasks', 'User', '$ro
        Tasks.updateTask(task._id, task);
      };
 
-    $scope.checkGroupAccess = function (group) {
+    $scope.checkGroupAccess = function (group, sup) {
```

Comment:
Can `sup` be given a more meaningful name? I can't work out from the code what it's for so a longer name might help.